calibration/CTDMOG/CGINS-CTDMOG-13597__20170402.csv

```diff
@@ -0,0 +1,25 @@
+serial,name,value,notes
+37-13597,CC_a0,-3.347561e-04,source_file 3305-00101-00528-A_SN_37-13597_QCT_Results_CTDMO-G.log
+37-13597,CC_a1,3.609641e-04,
+37-13597,CC_a2,-8.722234e-06,
+37-13597,CC_a3,3.151545e-07,
+37-13597,CC_cpcor,-9.57E-08,
+37-13597,CC_ctcor,3.25E-06,
+37-13597,CC_g,-9.797279e-01,
+37-13597,CC_h,1.344137e-01,
+37-13597,CC_i,-1.436922e-04,
+37-13597,CC_j,2.885123e-05,
+37-13597,CC_p_range,1450,
+37-13597,CC_pa0,9.065132e-01,
+37-13597,CC_pa1,4.622508e-03,
+37-13597,CC_pa2,-3.432140e-11,
+37-13597,CC_ptca0,5.247297e+05,
+37-13597,CC_ptca1,1.050053e+00,
+37-13597,CC_ptca2,3.487598e-01,
+37-13597,CC_ptcb0,2.495413e+001,
+37-13597,CC_ptcb1,2.500000e-005,
+37-13597,CC_ptcb2,0.00E+00,
+37-13597,CC_ptempa0,1.508106e+02,
+37-13597,CC_ptempa1,-7.036958e-02,
+37-13597,CC_ptempa2,8.378095e-07,
+37-13597,CC_wbotc,1.511425e-06,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT